### ISSUE TITLE
[KratosCore] Calculate of element size for quadratic elements

### ISSUE THREAD

**Issue by jgonzalezusua** [KratosMultiphysics/Kratos#10059]
**Description**
I am working in a flow in porous media formulation and to validate it, I will use quadratic elements. My problem is that, when I use the `FluidDynamicsApplication` which calls the `element_size_calculator.cpp`, there is no support for quadratic elements, so it throws me an compilation error. Would it be possible to include `MinimumElementSize` and `AverageElementSize` for quadratic elements (even in a simple version)? 
Right now, it is not possible to use quadratic elements in the `FluidDynamicsApplication` because of this limitation

**Scope**
- KratosCore
- Feature

**Comment by rubenzorrilla** [comment]
Agree. It would be very nice. @jgonzalezusua I already saw your PR (see my comments in there). Aside of the minor comments in there I'm very positive about this. As you suggest, we can start with a kind of simplified approach (as you did) to enable the use of quadratic elements.
